src/Sarif/FileSystem.cs

```diff
@@ -239,5 +239,32 @@ public void SetAttributes(string path, FileAttributes fileAttributes)
         {
             File.SetAttributes(path, fileAttributes);
         }
+
+        /// <summary>
+        /// Creates all directories and subdirectories in the specified path unless they
+        /// already exist.
+        /// </summary>
+        /// <param name="path">
+        /// The directory to create.
+        /// </param>
+        /// <returns>
+        /// An object that represents the directory at the specified path. This object is
+        /// returned regardless of whether a directory at the specified path already exists.
+        /// </returns>  
+        public DirectoryInfo CreateDirectory(string path)
+        {
+            return Directory.CreateDirectory(path);
+        }
+
+        /// <summary>
+        /// Deletes an empty directory from a specified path.
+        /// </summary>
+        /// <param name="path">
+        /// The name of the empty directory to remove. This directory must be writable and empty.
+        /// </param>
+        public void DeleteDirectory(string path, bool recursive = false)
+        {
+            Directory.Delete(path);
```

Comment:
>Directory.Delete(path); [](start = 12, length = 23)

Did you intend to use the 'recursive' input parameter?

Reply:
Oops :-)

---
In reply to: [296901139](https://github.com/microsoft/sarif-sdk/pull/1542#discussion_r296901139) [](ancestors = 296901139)